lib/models/visualisation.js

```diff
@@ -90,6 +90,8 @@ const schema = new mongoose.Schema({
   statementContents: [statementContents],
   hasBeenMigrated: { type: Boolean, default: false },
   timezone: { type: String },
+  showStatsAtTop: { type: Boolean, default: true },
+  showStatsAtBottom: { type: Boolean, default: true },
```

Comment:
Is it correct that by default, stats are shown at both top and bottom instead of either/or ? i.e. perhaps only one of the initialisers should be true.

Reply:
@lydiamros thank you for reviewing.

@Ian247 Are stats displayed at bottom or top, or both of them?

Reply:
It should be a toggle in the options to show stats at the top or bottom.

Reply:
As a result of discussed with @Ian247, we decided to put `show totals` toggle and `top / bottom` toggle. So stats at top or bottom is exclusive.